src/components/VariantSearchField.tsx

```diff
@@ -87,9 +88,15 @@ type Props = {
   onVariantSelect: (selection: VariantSelector) => void;
   isSimple: boolean;
   triggerSearch: () => void;
+  analysisMode?: AnalysisMode;
```

Comment:
After your edits, there is now no need to pass `analysisMode` into `VariantSearchField` anymore, is it?